packages/expo-av/ios/EXAV/EXAV+AudioSampleCallback.mm

```diff
@@ -0,0 +1,86 @@
+// Copyright © 2021-present 650 Industries, Inc. (aka Expo)
+
```

Comment:
There is a lot of code duplication between android and ios, if that is all features that will be included in c++ then it's probably fine, but if there are any plans to add more it's better to extract common code

Reply:
I also tried to add a shared C++ code base but honestly there's not a lot of code that can be shared - a lot of the code is platform specific (JNI calls, ObjC code, ...). Do you notice anything specifically that can be shared between both platforms?